python/lsst/ts/rubintv/handlers/pages_helpers.py

```diff
@@ -57,3 +61,24 @@ async def get_admin(request: Request) -> dict | None:
         return {"username": username, "email": email}
 
     return None
+
+
+async def get_key_from_type_and_visit(
+    camera_name: str,
+    type: str,
+    visit: str,
+) -> str:
+    """Get the key from the type and visit.
+    e.g from type="calexp_mosaic" and visit="2025042200233" return
+    key="lsstcam/2025-04-22/calexp_mosaic/000233/lsstcam_calexp_mosaic_2025-04-22_000233"
+    """
+    try:
+        day_obs_no_hyphens = visit[:8]
+        day_obs = f"{day_obs_no_hyphens[:4]}-{day_obs_no_hyphens[4:6]}-{day_obs_no_hyphens[6:]}"
+        seq_num = f"{int(visit[8:]):06}"
```

Comment:
I see you are padding with 6 zeros here, whereas you are padding with 5 zeros in `tests/handlers/external_test.py`: https://github.com/lsst-ts/rubintv/pull/258/files#diff-1d1567c5debba2d53e75004c9732111467b922dc158a744a4b43c36e2973bce1R233. Is that expected?